test/tools/auth0/handlers/logStreams.test.ts

```diff
@@ -77,11 +113,13 @@ describe('#logStreams handler', () => {
           update: ({ id }, data) => {
             didUpdateFunctionGetCalled = true;
             const expectedValue = (() => {
-              const value = logStreams.find((logStream) => {
+              const value = mockLogStreams.find((logStream) => {
                 return logStream.id === id;
               });
-              delete value.id;
-              delete value.type;
+              delete value.id; // Not expecting ID in PATCH payload
+              delete value.type; // Not expecting type in PATCH payload
+              //@ts-ignore because it's actually ok for status property to be omitted in PATCH payload
+              if (value?.status === 'suspended') delete value.status; // Not expecting status in PATCH payload if suspended
```

Comment:
This is the major addition here. I'm not in love with this test, it's more testing the implementation than behavior. But at least it's an extra assertion to make 🤷‍♂️ . This is why I add it to the E2E test for more security.